proto/src/packets/player_action.rs

```diff
@@ -0,0 +1,13 @@
+use bedrockrs_core::int::{VAR, LE};
+use bedrockrs_proto_derive::ProtoCodec;
+use crate::types::network_block_pos::NetworkBlockPos;
+use bedrockrs_shared::actor_runtime_id::ActorRuntimeID;
+
+#[derive(ProtoCodec, Debug, Clone)]
+pub struct PlayerActionPacket {
+    pub player_runtime_id: ActorRuntimeID,
+    pub action: VAR<i32>,
```

Comment:
could you turn this into an enum?

Reply:
Idk what the fields r, if ya were about to push this, maybe just close pr for now and u push ur stuff

Reply:
I am working on a similar packet, just confused them...
tho here are the possible type:
![image](https://github.com/user-attachments/assets/e3003ec3-27a9-4f93-825b-384b3b3371dd)
you can create a simple enum with them and use the `ProtoCodec` derive macro